src/main/java/software/amazon/awssdk/crt/mqtt/QoS.java

```diff
@@ -0,0 +1,35 @@
+package software.amazon.awssdk.crt.mqtt;
+
+/**
+ * Quality of Service associated with a publish action or subscription [MQTT-4.3].
+  */
+public enum QoS {
```

Comment:
Matthew suggested we name this "Qos" or "QualityOfService"

Reply:
Can do.